tools/pythonpkg/tests/fast/spark/test_spark_dataframe_sort.py

```diff
@@ -59,9 +64,16 @@ def test_sort_empty_params(self, spark):
         with pytest.raises(PySparkValueError):
             df = df.sort()
 
+    # See https://github.com/apache/spark/commit/0193d0f88a953063c41c41042fb58bd0badc155c
+    # for the PR which added that error to PySpark
+    @pytest.mark.skipif(
+        USE_ACTUAL_SPARK, reason="PySparkIndexError is only introduced in PySpark 4.0.0 which is not yet available"
```

Comment:
We could use `hasattr` just so it switches to being tested when PySpark 4.0.0 releases
similar to this:
```py
@pytest.mark.skipif(not hasattr(pa, 'ListViewArray'), reason='The pyarrow version does not support ListViewArrays')
```

Reply:
Good idea!